flavors/linuxlocal/local_flavor.go

```diff
@@ -33,7 +33,8 @@ type Flavor struct {
 	injected         bool
 	Logrus           logrus.Plugin
 	LinuxLocalClient localclient.Plugin
-	HTTP             httpmux.Plugin
+	RestHTTP     httpmux.Plugin
```

Comment:
The local&linuxlocal flavours do not need HTTP at all (I am not sure how would the statuscheck work).

1. Please provide pull request for vpp flavour.

2. NICE-to-HAVE: sync with Milan about local* flavours vs. HTTP vs. status check (to avoid nil seg faults). Ideally put if != nil for statuscheck.HTTP and remove HTTP fields from localflavour